src/docs/framework-integration/angular.md

```diff
@@ -10,75 +10,127 @@ contributors:
-        └── component-library/
-            └── src/
-                ├── lib/
-                └── public-api.ts
+└── packages
+    ├── stencil-library/
+    │   ├── stencil.config.js
+    │   └── src/components
+    └── angular-workspace/
+        └── projects/
+            └── component-library/
+                └── src/
+                    ├── lib/
+                    └── public-api.ts
+```
+
+This guide uses Lerna for the monorepo, but you can use other solutions such as Nx, TurboRepo, etc.
+
+To use Lerna with this walk through, globally install Lerna:
+
+```bash npm2yarn
+npm install --global lerna
+```
+
+#### Creating a Monorepo
+
+:::note
+If you already have a monorepo, skip this section.
+:::
+
+```bash npm2yarn
+# From your top-most-directory/, initialize a workspace
+lerna init
+
+# install dependencies
+npm install
+
+# install typescript and node types
+npm install typescript @types/node --save-dev
+```
+
+#### Creating a Stencil Component Library
+
+:::note
+If you already have a Stencil component library, skip this section.
+:::
+
+In the `packages/` directory, run the following commands to generate a Stencil component library:
+
+```bash npm2yarn
+npm init stencil components stencil-library
+cd stencil-library
+# Install dependencies
+npm install
 ```
 
 #### Creating an Angular Component Library
 
-> If you already have an Angular component library, skip this section.
+:::note
+If you already have an Angular component library, skip this section.
+:::
 
 The first time you want to create the component wrappers, you will need to have an Angular library package to write to.
 
-Using Angular's CLI, generate a workspace and a library for your Angular component wrappers:
+In the `packages/` directory, use the Angular CLI to generate a workspace and a library for your Angular component wrappers:
 
 ```bash
 npx -p @angular/cli ng new angular-workspace --no-create-application
 cd angular-workspace
 npx -p @angular/cli ng generate library component-library
 ```
 
-#### Creating a Stencil Component Library
+You can delete the `component-library.component.ts`, `component-library.service.ts`, and `*.spec.ts` files.
 
-> If you already have a Stencil component library, skip this section.
+You will also need to add your generated Stencil library as a dependency so import references can be resolved correctly:
 
-```bash
-npm init stencil components stencil-library
-cd stencil-library
-# Install dependencies
-npm install
-# or if using yarn
-yarn install
+```json
+// packages/angular-workspace/projects/component-library/package.json
+
+"dependencies": {
+  "stencil-library": "*"
+}
 ```
 
-### Adding Angular Output Target
+For more information, see the Lerna documentation on [package dependency management](https://lerna.js.org/docs/getting-started#package-dependency-management).
+
+:::note
+The Angular CLI will install Jasmine as a dependency to your Angular workspace. However, Stencil uses Jest as it's unit testing solution. To avoid
```

Comment:
Can we turn this into an `npm uninstall` command please? That way, we can use `npm2yarn` when this gets rebased (giving us the yarn equivalent more or less for free)